src/qt/trafficgraphwidget.h

```diff
@@ -22,7 +24,7 @@ class TrafficGraphWidget : public QWidget
 public:
     explicit TrafficGraphWidget(QWidget *parent = nullptr);
     void setClientModel(ClientModel *model);
-    int getGraphRangeMins() const;
+    std::chrono::minutes getGraphRangeMins() const;
```

Comment:
```suggestion
    std::chrono::minutes getGraphRange() const;
```

Also s/`setGraphRangeMins`/`setGraphRange`/

Reply:
Addressed in 9bb48bf657c7f2d36309807fbc7b55d0ab581a76